website/docs/docs/core/connect-data-platform/risingwave-setup.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "RisingWave setup"
+id: "risingwave-setup"
+description: "Read this guide to learn about how to set up RisingWave in dbt."
+meta:
+  maintained_by: RisingWave
+  pypi_package: 'dbt-risingwave'
+  authors: 'Dylan Chen'
+  github_repo: 'risingwavelabs/dbt-risingwave'
+  min_core_version: 'v1.6.1'
+  min_supported_version: ''
+  cloud_support: Not Supported
+  slack_channel_name: '#all-things-risingwave'
+  slack_channel_link: 'https://www.risingwave.com/slack'
+  platform_name: 'RisingWave'
+ config_page: '/reference/resource-configs/no-configs'
```

Comment:
The `config_page` frontmatter line's indentation is off here causing the build to fail. I believe it needs to be aligned with `platform_name` and the rest of the frontmatter meta properties above.

Reply:
I introduced that typo because I used the GitHub UI to make a code sugg. That should be fixed now.

Reply:
@runleonarun No worries! Looks like the build went through successfully after your edit 😁